src/lib.rs

```diff
@@ -12,9 +12,9 @@ use drop::async_trait;
 use drop::crypto::hash::{hash, Digest, HashError};
 use drop::crypto::key::exchange::PublicKey;
 use drop::crypto::sign::{self, KeyPair};
-use drop::system::manager::Handle;
-use drop::system::sender::ConvertSender;
-use drop::system::{message, Message, Processor, SampleError, Sampler, Sender, SenderError};
+use drop::system::{
+    message, ConvertSender, Handle, Message, Processor, SampleError, Sampler, Sender, SenderError,
+};
```

Comment:
multiline? (I personally prefer it this way but consistency)

Reply:
rustfmt rules of max 120 char per line

Reply:
Haha, alright x)